dawgz/schedulers.py

```diff
@@ -412,6 +415,7 @@ async def exec(self, job: Job) -> Any:
         ## Settings
         settings = self.settings.copy()
         settings.update(job.settings)
+        settings.pop("simtasks", None)
```

Comment:
```suggestion
```

Reply:
This line was added to avoid including the simtasks dictionary item in the loop at L421. Not removing it adds it as a SBATCH parameter which causes the job submission to fail.

I did not want to affect the code too much, so simply added that pop call.